requirements_test_min.txt

```diff
@@ -1,4 +1,4 @@
--e .[testutils,spelling]
+.[testutils,spelling]
```

Comment:
It's convenient to be able to do editable install when developing locally though, could we keep this one maybe ?

Reply:
Is updating the contributor docs to specify that an okay compromise? (I'm not used to expecting requirements files to take care of that for me.)

Reply:
CI installs from this file, so we'd have to do a bigger refactor otherwise.

Reply:
You're right, fait enough